cypress/e2e/hub/execution-environments.cy.ts

```diff
@@ -197,11 +197,72 @@ describe('Execution Environment Details tab', () => {
   it('should not change readme after cancel edit', () => {
     cy.visit(`/execution-environments/${executionEnvironment.name}/`);
     cy.get('[data-cy="readme"]').within(() => {
-      cy.clickButton('Edit');
+      cy.containsBy('button', 'Edit').click();
       cy.getByDataCy('raw-markdown').clear().type('{enter}this should not be saved.');
       cy.contains('Preview').parent().contains('this should not be saved.');
-      cy.clickButton('Cancel');
+      cy.containsBy('button', 'Cancel').click();
     });
     cy.get('[data-cy="readme"]').contains('this should not be saved.').should('not.exist');
   });
+
+  it('should successfully sync execution environment from Docker registry', () => {
+    cy.createHubRemoteRegistry().then((remoteRegistry) => {
+      cy.createHubExecutionEnvironment({
+        executionEnvironment: {
+          include_tags: ['latest'],
+          registry: remoteRegistry.id,
+        },
+      }).then((executionEnvironment) => {
+        cy.syncRemoteExecutionEnvironment(executionEnvironment);
+        cy.deleteHubExecutionEnvironment(executionEnvironment).then(() => {
+          cy.deleteHubRemoteRegistry(remoteRegistry);
+        });
+      });
+    });
+  });
+});
+
+describe('Execution Environment Activity tab', () => {
```

Comment:
@jerabekjiri we still could delete those resources created during those tests here.  Every resource created by a certain test ideally it should be deleted.